--- Makefile.bullet	2010-10-29 18:02:59.939029230 +0200
+++ Makefile.bullet.new	2010-10-29 18:02:47.682526439 +0200
@@ -51,7 +51,7 @@
 
 
 installed: wiped $(SOURCE_DIR)/unpacked
-	cd $(SOURCE_DIR) && cmake -DCMAKE_INSTALL_PREFIX=$(CURDIR) -DCMAKE_CXX_FLAGS="-fPIC -DBT_USE_DOUBLE_PRECISION" -D BUILD_SHARED_LIBS=on . 
+	cd $(SOURCE_DIR) && cmake -DCMAKE_INSTALL_PREFIX=$(CURDIR) -DCMAKE_CXX_FLAGS="-fPIC -DBT_USE_DOUBLE_PRECISION" -D BUILD_SHARED_LIBS=on -DCMAKE_TOOLCHAIN_FILE=$(ROS_ROOT)/rostoolchain.cmake .
 	# Bullet appears not be parallel-make safe
 	#cd $(SOURCE_DIR) && make $(PARALLEL_JOBS) 
 	cd $(SOURCE_DIR) && make $(BULLET_TARGETS) 
